dask/dataframe/core.py

```diff
@@ -243,6 +243,9 @@ class _Frame(DaskMethodsMixin, OperatorMethodMixin):
         Values along which we partition our blocks on the index
     """
     def __init__(self, dsk, name, meta, divisions):
+        if len(divisions) < 2:  # no partitions
```

Comment:
Not saying this is wrong, but it's a bit surprising to see this in a Parquet refactor. Do you recall what prompted it?

Reply:
These lines were added in [PR#4336](https://github.com/dask/dask/pull/4336) to address cases where an empty dataframe/partition is read in (like `test_parquet.py::test_empty`).  Not sure if there is a more appropriate fix.